config.json.example

```diff
@@ -4,5 +4,8 @@
     "memberCountGuild": {
         "guild": "guild_id",
         "channel": "channel_id"
-    }
+    },
+    "runtimeFlags": [
+        "DISABLE_TRIGGERS"
+    ]
```

Comment:
Is there a way to make this a cli flag? I think i'd prefer making this something that can be disabled without needing to edit a file. Maybe using environment variables?

Reply:
Yeah I can. The only reason I didn't is so you don't have to pass the args every time

Reply:
Yeah I see that, I'm just envisioning us being forgetful and not changing that when we need to.

Reply:
Sometime soon I'm going to revisit this and implement running with CLI flags because I do believe that is the right approach